src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md

```diff
@@ -85,10 +100,12 @@ TODO: Show metrics specific configuration (i.e MetricReaderOptions).
 * `ExportProcessorType`: Whether the exporter should use [Batch or Simple
   exporting
   processor](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk.md#built-in-span-processors).
-  The default is Batch.
+  The default is Batch. **Note**: This option is only applicable to
```

Comment:
Add a section for `LogRecordExportProcessorOptions` as well?

Reply:
May need a bigger refactoring for this. This also include `MetricReaderOptions` part. Would like to keep it in separate PR if that is ok. For immediate usage, the link to testlogs.cs should work.